changelog/_unreleased/2021-12-05-fix-determination-of-product-id-and-name-of-ga-plugin.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Fix determination of product id and name of GA plugin
+issue: NEXT-17278
+author: Max
+author_email: max@swk-web.com
+author_github: @aragon999
+---
+# Storefront
+* Change the selector for the product id to `[itemtype="https://schema.org/Product"] meta[itemprop="productID"]` and for the product name to `[itemtype="https://schema.org/Product"] [itemprop="name"]` to also work for soldout products
```

Comment:
Please also mention the file/class where this happens. `app/storefront/src/plugin/google-analytics/events/view-item.event.js`

Reply:
Changed the changelog. Is it fine like this? Or is more information required?

Reply:
Thank you, I think it is good like this.